internal/host/validations_test.go

```diff
@@ -1704,4 +1704,119 @@ var _ = Describe("Validations test", func() {
 			})
 		}
 	})
+
+	Context("NoIpCollisions", func() {
+
+		var (
+			host       models.Host
+			cluster    common.Cluster
+			hostId     strfmt.UUID
+			infraEnvId strfmt.UUID
+		)
+
+		BeforeEach(func() {
+			cluster = hostutil.GenerateTestCluster(clusterID)
+			Expect(db.Create(&cluster).Error).ToNot(HaveOccurred())
+			hostId, infraEnvId = strfmt.UUID(uuid.New().String()), strfmt.UUID(uuid.New().String())
+			host = hostutil.GenerateTestHostByKind(hostId, infraEnvId, &clusterID, models.HostStatusDiscovering, models.HostKindHost, models.HostRoleMaster)
+			Expect(db.Create(&host).Error).ShouldNot(HaveOccurred())
+			host = hostutil.GetHostFromDB(*host.ID, host.InfraEnvID, db).Host
+		})
+
+		It("Should mark the validation as pending if the connectivity report is unavailable yet", func() {
+			host.Connectivity = ""
+			mockAndRefreshStatus(&host)
+			host = hostutil.GetHostFromDB(*host.ID, host.InfraEnvID, db).Host
+			status, message, ok := getValidationResult(host.ValidationsInfo, NoIPCollisions)
+			Expect(ok).To(BeTrue())
```

Comment:
Lets compare this after `message` and `status` so that we'd get more info about the failure?

Reply:
That's fair, I'll reorder these statements